app/src/main/java/org/fossasia/susi/ai/data/SkillDetailsModel.kt

```diff
@@ -112,6 +115,27 @@ class SkillDetailsModel : ISkillDetailsModel {
         })
     }
 
+    override fun sendReport(queryObject: ReportSkill, listener: ISkillDetailsModel.OnReportSendListener) {
+        val map: MutableMap<String, String> = HashMap()
+        map.put("access_token", queryObject.accessToken)
+        map.put("feedback", queryObject.feedback)
+        map.put("skill", queryObject.skill)
+        map.put("group", queryObject.group)
+        map.put("model", queryObject.model)
+        reportSkillResponseCall = ClientBuilder.getSusiApi().reportSkill(map)
```

Comment:
Add a static method in ClientBuilder.java and pass an object to that method from here. In the defined method, create a map and return a ```Call<ReportSkillResponse>```.